src/main/java/com/inventory/structures/StackStruct.java

```diff
@@ -4,64 +4,88 @@
 import com.inventory.InventoryApplication;
 import com.inventory.repository.CategoryRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-
 //import java.util.Scanner;
+import java.sql.Connection;
```

Comment:
all sql connections should removed. they are not needed here

Reply:
okay